### ISSUE TITLE
Image Delete Icon Inconsistent With Table Remove Icon

### ISSUE THREAD

**Issue by duracell80** [liferay/alloy-editor#1288]
Table remove icon is a trash can, remove image icon is an x in a circle. A consistent icon for this action would help user familiarity and interaction.

**Comment by wincent** [comment]
I agree, @duracell80, consistency would be good here.

The trash can is probably not a good fit: in my experience it is used for higher-level entities that can be considered "object-like" — in other words "resources" that you may wish to dispose of — and it brings with it a bunch of other conceptual baggage such as the notion that you are transferring the resource to a temporary holding area (ie. the trash) from where you can restore it, or definitively delete it.

For something like a table, the x-in-a-circle sounds like a better fit.

@julien: what do you think?

**Comment by julien** [comment]
Yes, I also agree that we should use the same icons for both the table and the image.

I'd also rather use the "x-in-a-circle" icon because (in my mind) I usually associate the trash icon with deleting a file from my computer.

**Comment by jbalsas** [comment]
@drakonux, could you advise what is our convention here?

**Comment by drakonux** [comment]
Of course @jbalsas . The convention can be read in the [actions definition](https://liferay.design/lexicon/core-components/actions-definition/) section.

Anyway, it's important to see the context of the examples you mentioned. Please add images and/or steps to reproduce each scenario.

PS. In DXP the recycle bin can be disabled, so when that happens all the "recycle bin" icons should change to "x-in-a-circle".
